### ISSUE TITLE
Battleground map update while dead

### ISSUE THREAD

**Issue by Johanes022** [dalaranwow/dalaran-wow#5121]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: While you are dead in battleground (in ghost form) map doesnt update raid current position/movement and it stays as it was before you died, aka dots on the map do not move.When you ressurect map updates itself and works as normal.

**Expected behaviour**: When you open your map while beeing dead it should update other player position/movement as when you were alive

**Steps to reproduce the problem**:

1. Die in battleground
2. While beenig dead open map and observe other players on it and you will notice that they are just standing in place.
3. When you ressurect , map updates players current position and will track them as when you were alive.

**Comment by Aridam** [comment]
Yeah, Map/Minimap should actively-update that even while being a spirit.

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Johanes022.
Does this issue happens only in BGs or in all kind of instances like arenas/dungeons/raids?

**Comment by Aridam** [comment]
Coming back to this to update: This is basically related to View Distance and how on this server, it's shorter than what it should be -- because of that, anything beyond the View Distance _you_ have in relation to where you are (for example, dead, waiting at the Graveyard) won't be updated properly; meaning Maps, Unit Frames, and so on.

This is not _just_ affecting BGs -- this happens in general.
